File: src/main/kotlin/java/com/huanli233/xtcdecrypter/utils/XTCKeyUtil.java
package com.huanli233.xtcdecrypter.utils;

import java.util.regex.Matcher;
import java.util.regex.Pattern;

public class XTCKeyUtil {
	public static boolean checkKey(String key) {
		String regex = "^[a-zA-Z0-9]{32}:[a-zA-Z0-9=+/]{128}$";
        Pattern pattern = Pattern.compile(regex);
        Matcher matcher = pattern.matcher(key);
        return matcher.matches();
	}
}

File: src/main/kotlin/com/huanli233/xtcinfomanager/utils/XTCKeyUtil.kt
package java.com.huanli233.xtcdecrypter.utils

import java.util.regex.Pattern

object XTCKeyUtil {
    fun checkKey(key: String?): Boolean {
        val regex = "^[a-zA-Z0-9]{32}:[a-zA-Z0-9=+/]{128}$"
        val pattern = Pattern.compile(regex)
        val matcher = pattern.matcher(key)
        return matcher.matches()
    }
}
